assets/js/modules/analytics/util/index.js

```diff
@@ -286,6 +286,39 @@ export const extractAnalyticsDashboardData = ( reports, selectedStats, days ) =>
 	return dataMap;
 };
 
+/**
+ * Extracts the data required from an analytics 'site-analytics' request for an Area chart.
+ *
+ * @since n.e.x.t
+ *
+ * @param {Object} reports The data returned from the Analytics API call.
+ * @return {Array} Required data from 'site-analytics' request.
+ */
+export const extractAnalyticsChartData = ( reports ) => {
+	if ( ! reports || ! reports.length ) {
+		return null;
+	}
+
+	return [
+		[
+			{ type: 'date', label: 'Day' },
+			{ type: 'number', label: '' },
+			{ type: 'number', label: 'Users' },
+		],
+		...reports[ 0 ].data.rows.map( ( row ) => {
+			const { values } = row.metrics[ 0 ];
+			const dateString = row.dimensions[ 0 ];
+			const date = parseDimensionStringToDate( dateString );
+
+			return [
+				date,
+				null,
+				values[ 0 ],
+			];
+		} ),
+	];
+};
```

Comment:
Let's keep this inline (or in a privately used function) within `UserCountGraph` for now. The name here is very broad for that the function does something fairly specific. We only require it in one place for now, so let's move this as an internal function into `UserCountGraph.js`. We can reconsider reusing/improving/abstracting it when this becomes necessary elsewhere.

Normally we might be able to do that as part of this PR and invest a bit more time, but since it's late, let's go with the above for now.